src/styles/blocks/header.scss

```diff
@@ -0,0 +1,71 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: sticky;
+  top: 0;
+  padding: 0 50px;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
+  @include widthHeight(40px, 40px);
+
+  &__image {
+    @include widthHeight(100%, 100%);
+  }
+}
+
+.nav {
+  display: block;
```

Comment:
I don't think you need this. nav tag is a block element.

Reply:
I need it to wrap every elements inside of it 😀